### PR TITLE
kernel: catch up rxrpc

### PR DESCRIPTION
**Pull request by nwf**
- rxrpc: New dependencies required for the module to be useful, and correct IPv6 support
- crypto-misc: remove fcrypt from miscellaneous, as it has its own dedicated knob

### CODE REVIEW

**Comment by nwf** [comment]
I believe the build failure is not mine, just a glitch in CI.

File: package/kernel/linux/modules/netsupport.mk [hunk 1]
```diff
@@ -1297,13 +1297,21 @@ define KernelPackage/rxrpc
...
+	CONFIG_AF_RXRPC_IPV6=y \
+	CONFIG_RXKAD \
 	CONFIG_AF_RXRPC_DEBUG=n
   FILES:= \
 	$(LINUX_DIR)/net/rxrpc/rxrpc.ko
-  AUTOLOAD:=$(call AutoLoad,30,rxrpc.ko)
-  DEPENDS:= +kmod-crypto-manager +kmod-crypto-pcbc +kmod-crypto-fcrypt \
-    +kmod-udptunnel4 +kmod-udptunnel6
+  AUTOLOAD:=$(call AutoLoad,30,rxrpc)
+  DEPENDS:= \
```
Comment (robimarko) [inline package/kernel/linux/modules/netsupport.mk:1306]: Are these kernel version dependent?

Comment (nwf) [reply package/kernel/linux/modules/netsupport.mk:1306]: I don't believe so, no.  Some of these are not required in the sense of linkage dependencies (I believe only the `udptunnel` ones are strictly required), but are required in order to make authenticated and encrypted connections (the "rxkad" cryptosystem involves hmac-md5 and fcrypt-pcbc; see https://datatracker.ietf.org/doc/html/draft-kaduk-afs3-rxkad-k5-kdf-00).

Comment (nwf) [reply package/kernel/linux/modules/netsupport.mk:1306]: If you'd prefer that `crypto-user` weren't a dependency here but rather were a dependency of the userspace program(s) to use rxrpc, I could do that instead.

Comment (robimarko) [reply package/kernel/linux/modules/netsupport.mk:1306]: That would be a good idea, do that and I will merge it

Comment (nwf) [reply package/kernel/linux/modules/netsupport.mk:1306]: Great, done.  Thanks!

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!

**Pull request merged**